### PR TITLE
Changes GitHub to Guild.xyz

### PR DESCRIPTION
**Pull request by SoNiC-HeRE**
## Related Issue

- Information about the related issue

Closes: #344 

### Describe the changes you've made
Updated the url to https://guild.xyz/webx-dao , changed logo as mentioned in the comment and changed the text to Guild.xyz

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)
Before:
image

After: 
image

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by darkterminal** [comment]
OhMyPunkGuild!!! :fire: :fire_engine: :fire: :fire: That's gwrreeet!!!

File: app/page.tsx [hunk 1]
```diff
@@ -35,11 +35,11 @@ export default function IndexPage() {
             <Link
               target="_blank"
               rel="noreferrer"
-              href={siteConfig.links.webxdao_gh}
+              href="https://guild.xyz/webx-dao"
               className={buttonVariants({ variant: "outline", size: "lg" })}
             >
-              <Icons.gitHub className="mr-2 h-5 w-5" />
-              GitHub
+              <img src="/guild-logo.png" className="mr-2 h-5 w-5" />
```
Comment (darkterminal) [inline app/page.tsx:41]: This :european_castle: is a-punk-wesome!!!

File: app/page.tsx [hunk 2]
```diff
@@ -35,11 +35,11 @@ export default function IndexPage() {
             <Link
               target="_blank"
               rel="noreferrer"
-              href={siteConfig.links.webxdao_gh}
+              href="https://guild.xyz/webx-dao"
```
Comment (Mustafiz04) [inline app/page.tsx:38]: can you please move this hardcode url to the site config file?

**Review by darkterminal — APPROVED** [review]

File: app/page.tsx (hunk 1 above)
Comment (Mustafiz04) [inline app/page.tsx:41]: please use `<Icons.guild className="mr-2 h-5 w-5" />`

File: app/page.tsx (hunk 2 above)
Comment (darkterminal) [reply app/page.tsx:38]: Ah... yeah, I missed this. I just :scream: with the :european_castle:

**Review by Mustafiz04 — CHANGES_REQUESTED** [review]
Please fix the changes

File: app/page.tsx (hunk 1 above)
Comment (SoNiC-HeRE) [reply app/page.tsx:41]: okay

**Comment by vinzvinci** [comment]
Any updates on this? or need some help?

**Comment by SoNiC-HeRE** [comment]
> Any updates on this? or need some help?

Yeah I've implemented the above changes: updated the link to siteconfig and also implemented a rounded full image but am figuring out how to add an image to a group of objects: ```Icons.guild``` cause rendering it directly gives me error

**Comment by mkubdev** [comment]
Hello, thanks you for your contribution!  🚀 Your PR still requires changes from @Mustafiz04.

1. The link for `guild.xyz` line 42 in `app/pages.tsx` should not be hardcoded. Kindly ask you to add this in `config/site/ts` :
```ts
// config/site.ts

...

 links: {
    //twitter: "https://twitter.com/WebXDAO",
    //github: "https://github.com/WebXDAO",
    webxdao_gh: "https://github.com/WebXDAO/",
    webxdao_getstarted: "https://github.com/WebXDAO/start-here",
    webxdao_discord: "https://discord.webxdao.xyz/",
    webxdao_twitter: "https://twitter.com/WebXDAO",
    webxdao_linkedin: "https://www.linkedin.com/company/webxdao/",
    
    // Add this 👇
    webxdao_guild: "https://guild.xyz/webx-dao"
  },
  ```
  2. Don't use `` inside a Next.js application. You need to use [`<Image />`](https://nextjs.org/docs/pages/api-reference/components/image) component.
  
  > Icons.guild cause rendering it directly gives me error
  
  Can you share your error?
  
  This should work with this snippet:
  
  ```html
   <div className="flex justify-center gap-4">
            <Link
              href={siteConfig.links.webxdao_getstarted}
              target="_blank"
              rel="noreferrer"
              className={buttonVariants({ size: "lg" })}
            >
              Get started
            </Link>
            <Link
              target="_blank"
              rel="noreferrer"
              href={siteConfig.links.webxdao_guild}
              className={buttonVariants({ variant: "outline", size: "lg" })}
            >
              <Image alt="guild.xyz logo" src="/guild-logo.png" className="mr-2 h-5 w-5" width={150} height={150}/>
                Guild.xyz
            </Link>
</div>
```

**Comment by SoNiC-HeRE** [comment]
Thanks for clarifiying
I already implemented the link part but what i was trying to do was to add an image and then export it in the ```Icons.tsx``` file in order to use ```Icons.guild``` and was faced by an error ```Usnupported Server Component```

P.S: Implemented the changes

**Review by mkubdev — APPROVED** [review]
Well done! Thanks a lot for your contribution, everything looks good to me. 👌⚡

**Comment by mkubdev** [comment]
> Thanks for clarifiying I already implemented the link part but what i was trying to do was to add an image to group of object under Icons and then export it in the `Icons.tsx` file in order to use `Icons.guild` and was faced by an error `Unsupported Server Component` Was confused with this [#346 (comment)](https://github.com/WebXDAO/WebXDAO.github.io/pull/346#discussion_r1204400816)
> 
> P.S: Implemented the changes

Yes, I'm having issues too with using /public image inside the Icons component... Thanks for trying! :)

**Review by vinzvinci — APPROVED** [review]
LGTM! Thanks for the help @mkubdev 🎉

**Pull request merged**